sevsnpmeasure/cmd/root.go

```diff
@@ -7,23 +7,156 @@ SPDX-License-Identifier: Apache-2.0
 package cmd
 
 import (
+	"encoding/hex"
+	"fmt"
+	"os"
+
 	"github.com/spf13/cobra"
+	"github.com/virtee/sev-snp-measure-go/cpuid"
+	"github.com/virtee/sev-snp-measure-go/guest"
+	"github.com/virtee/sev-snp-measure-go/vmmtypes"
+)
+
+const defGuestFeatures = 0x1
+
+var (
+	mode          string
+	vcpus         int
+	vcpuType      string
+	ovmfFile      string
+	kernelFile    string
+	initrdFile    string
+	append        string
+	vcpuSig       int
+	vcpuFamily    int
+	vcpuModel     int
+	vcpuStepping  int
+	vmmtype       string
+	guestFeatures uint64
+	varsSize      int64
+	varsFile      string
+	snpOvmfHash   string
+	dumpVmsa      bool
+	svsmFile      string
 )
 
 // Execute executes the root command.
 func Execute() error {
-	return newRootCmd().Execute()
+	return NewRootCmd().Execute()
 }
 
 // newRootCmd creates the root command.
-func newRootCmd() *cobra.Command {
+func NewRootCmd() *cobra.Command {
```

Comment:
```suggestion
func newRootCmd() *cobra.Command {
```
is there a reason for exporting this function?

Reply:
I wanted to use it in another CLI tool, but I have kept it unexported now